File: project 2/bmi.c
#include <stdio.h>

int main()
{
    printf("* BMI Calculator *\n");
    char calculate;
    do
    {
        int weight,height;
        int BMI;
        printf("Enter a weight: ");
        scanf("%d", &weight);
        printf("Enter a height: ");
        scanf("%d", &height);

        BMI = weight/(height*height);
        printf("%d\n",BMI);
        
        printf("\nDo you want to calculate again? (y/n):");
        scanf(" %c", &calculate);
    }

    while (calculate == 'y' || calculate == 'Y');
    printf("Thank you.\n");
}

File: project 2/bmi.py
print("* BMI Calculator *")

while True:
    weight=int(input("Enter a weight: "))
    height=int(input("Enter a height: "))
    bmi = weight/(height*height)
    print(bmi)
    
    calculate = input("\nDo you want to calculate again? (y/n): ").lower()
    if calculate != "y":
        print("Thank you.")
        break
